api-reference/50 Common/Object Structures/positionConfig/boundaryOffset/boundaryOffset.md

```diff
@@ -4,25 +4,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies the horizontal and vertical offset from the window's boundaries.
+Specifies the offset of the boundaries from the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) element.
```

Comment:
Is there a reason you deleted "horizontal and vertical" in the new version?

Reply:
Can we say:
Specifies the offset of a boundary from the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) element.